dashboard/src/components/views/EnvironmentView/EntitiesContent.js

```diff
@@ -70,8 +70,9 @@ class EntitiesContent extends React.PureComponent {
                 />
               </Content>
               <EntitiesList
-                loading={loading || aborted}
                 environment={environment}
+                loading={loading || aborted}
+                onQueryChange={this.props.setQueryParams}
```

Comment:
Use `onChangeQuery` for consistency with other event handler props: `on{EventName}{TargetName}`